### ISSUE TITLE
Commands Api (command computers) require a setting meant for command Api (peripheral wrapping to vanilla command block)

### ISSUE THREAD

**Issue by KMorris29** [cc-tweaked/cc-restitched#23]
## Useful information to include:
 - 1.17,1
 - CC: Restitched version 1.92.2
 - No Logs because it isn't a logged event.
 
 - Detailed reproduction steps: When starting a fresh instance with just CC:R installed, and starting a creative world, command computers cannot use the commands api (and are therefore useless except as regular computers). However, when switching the per-world config of command_block_enabled, and then restarting the world, the commands global is suddenly usable.
 
 - Expected behavior: As the command computers require the user to be in creative and opped anyway (unless the config command_require_creative is changed to false), they are expected to work without changing the configs within the world. The command block api is slightly different, it works with any computer next to a command block (I think?), and definitely needs a config option to default to false to avoid messing with adventure maps and the like.

-Likely cause: the api's are called commands, with an s, and command, no s, respectively, so I'm guessing that wherever the config is set a one-letter typo happened.

**Comment by KMorris29** [comment]
Actually it is a little worse than I thought: whenever the computer is broken and replaced, even if it has the same ID as the computer that was originally working, the world needs to be restarted once again.

**Comment by Merith-TK** [comment]
Would you be willing to test this with me? send me a message over discord `Merith#5829`

**Comment by Merith-TK** [comment]
Testing concludes

1) Requires CommandBlock Peripheral config enabled to even function
2) Only Command Computers that existed on wold load are able to utilize the commands api
3) breaking the computer and placing it down prevents it from being able to use the commands api
4) if a perviously broken computer is reloaded when the world loads, it works just fine
  - `/reload` does not fix this, the whole world has to be exited and re-opened

TLDR: **Command Computers are fucked**
